src/Data/Edge.cpp

```diff
@@ -390,7 +390,7 @@ osg::ref_ptr<osg::Geode> Data::Edge::createEdgeQuad( osg::StateSet* bbState )
 	nodeQuad->setStateSet( bbState );
 
 	osg::ref_ptr<osg::Geode> geode = new osg::Geode;
-	geode->setNodeMask( geode->getNodeMask() & static_cast<unsigned int>( ~0x2 ) );
+    //geode->setNodeMask( geode->getNodeMask() & static_cast<unsigned int>( ~0x2 ) );
```

Comment:
treba alebo netreba ???
ak netreba zmazat, 
ak mozno bude treba kvoli inym castiam kodu, tak to treba okomentovat